CHANGELOG.md

```diff
@@ -45,6 +45,16 @@ Changelog is rather internal in nature. See release notes for the public overvie
 
 <!-- Release notes prepared for all items below -->
 
+- [#453]
```

Comment:
Just a very minor note, could you please move it above `<!-- Release notes prepared for all items below -->` line? That's just so that we know at what point release notes ends. This should be temporary way of doing so, as soon as we have the first release ready.

Reply:
In any case, nice first item :) Looking great

Reply:
ah great point about the placement, @MisRob, thank you!